### PR TITLE
Resolve Component Governance bugs

### PR DESCRIPTION
**Pull request by wtgodbe**
Should resolve Component Governance bugs, except a couple that I'm following up on. Will want to take a submodule update once https://github.com/dotnet/spa-templates/pull/30 is in.

CC @Pilchie

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -226,11 +226,11 @@
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension60x64Version>$(MicrosoftAspNetCoreAzureAppServicesSiteExtension60Version)</MicrosoftAspNetCoreAzureAppServicesSiteExtension60x64Version>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension60x86Version>$(MicrosoftAspNetCoreAzureAppServicesSiteExtension60Version)</MicrosoftAspNetCoreAzureAppServicesSiteExtension60x86Version>
     <!-- 3rd party dependencies -->
-    <AngleSharpVersion>0.9.9</AngleSharpVersion>
+    <AngleSharpVersion>0.10.0</AngleSharpVersion>
```
Comment (wtgodbe) [inline eng/Versions.props:229]: Changes to this file are to prevent `System.Net.Http 4.1.0` from being brought in transitively

**Review by TanayParikh — APPROVED** [review]

**Comment by wtgodbe** [comment]
>   Error: require() of ES Module D:\a\_work\1\s\src\SignalR\clients\ts\common\node_modules\ansi-regex\index.js from D:\a\_work\1\s\src\SignalR\clients\ts\common\node_modules\strip-ansi\index.js not supported.

@javiercn @captainsafia @pranavkm @BrennanConroy `ansi-regex 5.0.0` was one of the components flagged by CompGov, but upgrading to 5.0.1 gives the above error - any ideas on what else I need to change?

**Comment by BrennanConroy** [comment]
Try upgrading to 5.0.1 instead of 6.0.1

**Comment by wtgodbe** [comment]
Hmm, looks like resolving `>= 5.0.1` is bringing in `6.0.1`. I'll ask for 5.0.1 exactly

**Review by Pilchie — APPROVED** [review]
Thanks!

**Review by BrennanConroy — APPROVED** [review]

**Pull request merged**